CVAE_testbed/datasets/quilt_aics_features.py

```diff
@@ -0,0 +1,239 @@
+import torch
+from quilt3 import Package
+from collections import defaultdict
+from torch.utils.data import Dataset
+import pandas as pd
+import os, json
+from featuredb import FeatureDatabase
+from sklearn import preprocessing
+import numpy as np
+from scipy import stats
+from scipy.stats import zscore
+import sklearn
+
+
+class QuiltAicsFeatures(Dataset):
+    def __init__(self, num_batches, BATCH_SIZE, model_kwargs, shuffle=True, corr=False, train=True, mask=False):
+        """
+        Args: 
+            num_batches: Number of batches of synthetic data
+            BATCH_SIZE: batchsize of synthetic data
+            model_kwargs: dictionary containing "x_dim" which indicates input data size
+            shuffle:  True sets condition vector in input data to 0 for all possible permutations
+            corr: True sets dependent input dimensions via a correlation matrix 
+        """
+        self.num_batches = num_batches
+        self.BATCH_SIZE = BATCH_SIZE
+        self.corr = corr
+        self.shuffle = shuffle
+        self.model_kwargs = model_kwargs
+        self.train = train
+
+        Batches_X_train, Batches_C_train, Batches_conds_train = torch.empty([0]), torch.empty([0]), torch.empty([0])
+        Batches_X_test, Batches_C_test, Batches_conds_test = torch.empty([0]), torch.empty([0]), torch.empty([0])
+
+        ds = Package.browse("aics/pipeline_integrated_single_cell", "s3://allencell")
```

Comment:
You should consider adding a `try` `except` for if the features have been downloaded or not. If the `json_quilt_path` doesn't exist, download them to that dir

Reply:
This would be in place of the default `try` `except` below.

Reply:
Good point! Will make change